src/index.js

```diff
@@ -47,6 +48,19 @@ class Dropzone extends React.Component {
     document.body.onfocus = this.onFileDialogCancel;
   }
 
+  componentWillReceiveProps(nextProps) {
+    // If the accept setting has changed, and component is currently in the accept or reject states
+    if (nextProps.accept !== this.props.accept && (this.state.isDragActive || this.state.isDragReject)) {
+      const allFilesAccepted = this.allFilesAccepted(this.draggedFiles, nextProps);
```

Comment:
By looking at this I think it would make more sense to put dragged file into state and calculate `isDragActive` and `isDragReject` in the `render` method since this is very cheap. Thoughts?